src/components/VideoScript/index.jsx

```diff
@@ -47,38 +52,100 @@ function VideoScript({
       }
     });
 
-    scriptRef.current[focusedIndex].scrollIntoView({
-      behavior: "smooth",
-      block: "start",
-      inline: "start",
-    });
-  }, [currentVideoTime]);
+    if (showTranscript) {
+      scriptRef.current[focusedIndex].scrollIntoView({
+        behavior: "smooth",
+        block: "start",
+        inline: "start",
+      });
+    }
+  }, [currentVideoTime, showTranscript]);
+
+  function handleShowTranscriptButton() {
+    setShowTranscript((prev) => !prev);
+    setShowLanguageDropdown(false);
+  }
+
+  function handleTranscriptLanguageButton() {
+    if (showTranscript) {
+      setShowLanguageDropdown((prev) => !prev);
+    }
+  }
+
+  function handleLanguageSelect(language) {
+    setTranscriptLanguage(language);
+    setShowLanguageDropdown(false);
+  }
 
   return (
-    <div className="absolute top-[176px] left-[830px] h-[610px] mr-4 p-2 border-2 border-slate-500 text-justify overflow-scroll">
-      {transcript !== " " &&
-        transcriptTimeLines.map((transcriptTimeLine, index) => {
-          const isInTime =
-            timeToSeconds(transcriptTimeLine) <= currentVideoTime &&
-            timeToSeconds(transcriptTimeLines[index + 1]) > currentVideoTime;
-
-          return (
-            <button
-              key={youtubeVideoId + transcriptTimeLine + transcripts[index]}
-              className={`${isInTime && "bg-red-100"} flex hover:bg-gray-100`}
-              onClick={() => {
-                seekToTime(timeToSeconds(transcriptTimeLine));
-              }}
-              ref={(element) => {
-                scriptRef.current[index] = element;
-              }}
-            >
-              <p className="w-12 mr-2">{transcriptTimeLine}</p>
-              <p className="w-48">{transcripts[index]}</p>
-              <p className="w-48">{koreanScripts[index]}</p>
-            </button>
-          );
-        })}
+    <div
+      className="overflow-y-auto w-[35rem] mr-4 mt-5 ml-10 border-2 border-grey-800 rounded-xl"
+      style={{ height: showTranscript ? playerDimensions.height : "" }}
+    >
+      <div className="flex items-center justify-between sticky z-10 p-2 top-0  bg-white font-bold text-2xl">
```

Comment:
띄어쓰기 하나 더 들어간 것 같습니다!
```suggestion
      <div className="flex items-center justify-between sticky z-10 p-2 top-0 bg-white font-bold text-2xl">
```